plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -394,8 +394,16 @@ private void prepareHost(HostMO hostMo, String privateTrafficLabel) throws Excep
             HypervisorHostHelper.prepareNetwork(vSwitchName, "cloud.private", hostMo, vlanId, null, null, 180000, false, BroadcastDomainType.Vlan, null, null);
         }
         else {
+            int portsPerDvPortGroup = _portsPerDvPortGroup;
+            AboutInfo about = hostMo.getHostAboutInfo();
+            if (about != null) {
+                String version = about.getApiVersion();
```

Comment:
Are the versions fixed? I mean, can't we have `4.2`, `4.3`, and so on.

Reply:
@rafaelweingartner No VMware ESXi/ESX hosts with these release versions. Take a look at the versions in the article https://kb.vmware.com/s/article/2143832.

Reply:
LGTM